File: app/src/main/java/com/deepquotes/utils/DBManager.java
package com.deepquotes.utils;

import android.content.Context;

import static com.deepquotes.utils.Constant.*;

public class DBManager {

    private static QuotesSQLHelper mQuotesSQLHelper;

    public static QuotesSQLHelper getInstance(Context context){
        if (mQuotesSQLHelper==null){
            mQuotesSQLHelper = new QuotesSQLHelper(context,DATABASE_NAME,null,DATABASE_VERSION);
            return mQuotesSQLHelper;
        }
        return mQuotesSQLHelper;
    }

}

File: app/src/main/java/com/deepquotes/utils/DBManager.kt
package com.deepquotes.utils

import android.content.Context
import com.deepquotes.utils.Constant.DATABASE_NAME
import com.deepquotes.utils.Constant.DATABASE_VERSION

object DBManager {
    private var mQuotesSQLHelper: QuotesSQLHelper? = null
    fun getInstance(context: Context?): QuotesSQLHelper? {
        if (mQuotesSQLHelper == null) {
            mQuotesSQLHelper = QuotesSQLHelper(context, DATABASE_NAME, null, DATABASE_VERSION)
            return mQuotesSQLHelper
        }
        return mQuotesSQLHelper
    }
}